jpademo/src/main/java/com/likelion/jpademo/entity/Post.java

```diff
@@ -0,0 +1,41 @@
+package com.likelion.jpademo.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@NoArgsConstructor
+public class Post {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String title;
+
+    @Lob
+    private String content;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id")
+    private Member member;
+
+    @Builder
+    public Post(String title, String content, Member member) {
+        this.title = title;
+        this.content = content;
+        this.member = member;
+    }
+
+    public void setMember(Member member) {
+        this.member = member;
+    } //이 부분도 삭제했더니 오류가 나서 찾아보니 member와의 연관관계를 위해서 필요하다고 해서 그냥 뒀는데 이게 남아있어도 오류를 야기할 수 있나요?
```

Comment:
```suggestion
    protected void setMember(Member member) {
        this.member = member;
    } 
```

연관관계 설정은 반드시 Member의 addPost 편의 메서드를 통해서만 이루어지도록 강제하여 객체지향적인 구조를 만들 수 있을 것 같아요!